keyvalue/server/src/main/java/alluxio/master/keyvalue/KeyValueMaster.java

```diff
@@ -60,6 +62,12 @@
  */
 @ThreadSafe
 public final class KeyValueMaster extends AbstractMaster {
+  private static final Set<Class<?>> DEPS = new HashSet<>();
```

Comment:
the type parameterization comes after the `.`, e.g. `ImmutableSet.<Class<?>>of()`